src/components/QueryRuleCustomData.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div v-if="state">
+    <slot :items="state.items">
+      <div v-for="(item, key) in state.items" :key="key">
+        <slot name="item" :item="item">
+          <pre>{{item}}</pre>
```

Comment:
We don't provide any default template in InstantSearch.js. Do we need to add one there?

Reply:
I found it really useful for debugging before committing to a certain way to display them, I implemented the same in Angular InstantSearch